units/Legion/Labs/legcorsy.lua

```diff
@@ -0,0 +1,3 @@
+local def = VFS.Include("units/CorBuildings/SeaFactories/corsy.lua").corsy
```

Comment:
i remember behe explicitly saying he doesn't want this style of defs (because external parsers become sad, plus this gets pwned by baking). @Beherith cc

Reply:
also **leg**ion **s**hip**y**ard should probably be called `legsy` and not `legcorsy`

Reply:
No opinion on what the final name should be, I did this as a proof of concent, and just called it `legcorsy` to make it clear that it was a duplicated unit

Reply:
So, after reading the discussion, my main issues with VFS.Include style defs stands, but if no better solution can be thought of, then I think they can be used as an intermediary for this specific case.